service/controller/key/key.go

```diff
@@ -203,6 +200,13 @@ func ClusterNamespace(customObject v1alpha1.KVMConfig) string {
 	return ClusterID(customObject)
 }
 
+func NodePodKey(cluster v1alpha1.KVMConfig, node corev1.Node) client.ObjectKey {
+	return client.ObjectKey{
```

Comment:
Used with the `ctrlClient` to get the key of the node pod corresponding to a given cluster and WC node.